ComPact.ConsumerTests/PactV3ConsumerTests.cs

```diff
@@ -194,6 +196,152 @@ public async Task ShouldWorkWithEmptyResponse()
             var pactFile = File.ReadAllText(pactDir + "V3-consumer-V3-provider.json");
             Assert.AreEqual("V3-consumer", JsonConvert.DeserializeObject<Contract>(pactFile).Consumer.Name);
             File.Delete(pactDir + "V3-consumer-V3-provider.json");
+        }
+
+        [TestMethod]
+        public async Task ShouldMatchPutRequest()
```

Comment:
I would replacing one, maybe even both of these with unit tests (Models.V3.RequestMatchTests). It's more focused and less duplicate code.

Reply:
Done